libs-rs/rust-toolchain

```diff
@@ -0,0 +1 @@
+nightly-2020-05-15
```

Comment:
Our contracts should be compilable with stable. What would be the reason to use nightly here?

Reply:
For intention to run cross contract test:
```
near-runtime-standalone = { git = "https://github.com/nearprotocol/nearcore.git" }
```
and nearcore need nightly to build. Maybe we should
- specify stable-xxx in rust-toolchain to produce determinstic build on contracts
- use script to switch to nightly to build cross contract test

Reply:
Oh actually @evgenykuzyakov added
```
RUSTFLAGS='-C link-arg=-s' cargo +stable build --target wasm32-unknown-unknown --release
```
in build.sh, this is how we did in core-contracts, so contracts are built with stable. but problem is we can't specify a determined rust version here